io-engine/src/bin/io-engine-client/v1/nexus_cli.rs

```diff
@@ -16,56 +16,71 @@ use uuid::Uuid;
 pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
     let create = SubCommand::with_name("create")
         .about("Create a new nexus device")
-        .arg(
-            Arg::with_name("name")
-                .required(true)
-                .index(1)
-                .help("name of the nexus"),
-        )
         .arg(
             Arg::with_name("uuid")
                 .required(true)
+                .index(1)
                 .help("uuid for the nexus, if uuid is not known please provide \"\" to autogenerate"),
         )
         .arg(
             Arg::with_name("size")
                 .required(true)
+                .index(2)
                 .help("size with optional unit suffix"),
         )
         .arg(
-            Arg::with_name("min-cntlid")
+            Arg::with_name("children")
                 .required(true)
+                .index(3)
+                .multiple(true)
+                .help("list of children to add"),
+        )
+        .arg(
+            Arg::with_name("name")
+                .required(false)
+                .long("name")
+                .help("name of the nexus"),
+        )
+        .arg(
+            Arg::with_name("min-cntlid")
+                .required(false)
+                .default_value("1")
+                .long("min-cntlid")
                 .help("minimum NVMe controller ID for sharing over NVMf"),
         )
         .arg(
             Arg::with_name("max-cntlid")
-                .required(true)
+                .required(false)
+                .default_value("65519")
+                .long("max-cntlid")
                 .help("maximum NVMe controller ID"),
         )
         .arg(
             Arg::with_name("resv-key")
-                .required(true)
+                .required(false)
+                .default_value("0")
+                .long("resv-key")
                 .help("NVMe reservation key for children"),
         )
         .arg(
             Arg::with_name("preempt-key")
-                .required(true)
+                .required(false)
+                .default_value("0")
+                .long("preempt-key")
                 .help("NVMe preempt key for children, 0 for no preemption"),
         )
-        .arg(
-            Arg::with_name("nexus-info-key")
-                .required(true)
-                .help("Key used to persist the NexusInfo structure to the persistent store"),
-        )
         .arg(Arg::with_name("resv-type")
-            .required(true)
+            .required(false)
+            .default_value("Reserved")
```

Comment:
I think default should be "" ? In dataplane code None will be interpreted to WriteExclusiveAllRegs.

Reply:
Done